WalletWasabi.Fluent/Controls/DialogContentHost.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Mixins;
+using Avalonia.Controls.Primitives;
+using Avalonia.Metadata;
+
+namespace WalletWasabi.Fluent.Controls
+{
+	/// <summary>
+	/// Container for NavBarItems.
```

Comment:
Fix description here.

Reply:
woops thanks bud, didn't noticed that lol